lib/linters/decimal_zero.js

```diff
@@ -9,58 +9,67 @@ module.exports = {
 
     lint: function decimalZeroLinter (config, node) {
         var number;
-        var value;
-        var output = {
-            inclusion: 'with',
-            type: null
-        };
+        var result = [];
+        var self = this;
 
-        node.forEach('value', function (element) {
-            value = element.first('dimension');
-            value = value && value.first('number');
-        });
```

Comment:
This logic would iterate over all "value" nodes, but the `value` variable would only be set to the last one in the iteration.